### PR TITLE
Simplifying documentation on how to contribute

### PR DESCRIPTION
**Pull request by karolikl**
- Cleaned up empty documentation pages
- Simplified our contributors guide

### CODE REVIEW

File: Documentation/Contribution/development_environment.md [hunk 1]
```diff
@@ -2,45 +2,44 @@
...
 
-[MongoChef / Studio3T](https://studio3t.com) is a great tool - supports cross platform.
-For using this with Azure and CosmosDB, there is a write-up [here](https://docs.microsoft.com/en-us/azure/cosmos-db/mongodb-mongochef)
+The Community edition of Visual Studio 2017 and Visual Studio 2017 for Mac can be used for free in open source development: 
+Download it [here](https://www.visualstudio.com/vs/community/).
 
 ## Terminal
 
-Any terminal and git. To get quickly up and running, suggested options are [cmder]([http://cmder.net/]) on Windows and [iTerm2](https://www.iterm2.com/downloads.html) on Mac.
+You will need any terminal and git. To get quickly up and running, suggested options are [cmder]([http://cmder.net/]) on Windows and [iTerm2](https://www.iterm2.com/downloads.html) on Mac.
```
Comment (anderaus) [inline Documentation/Contribution/development_environment.md]: Wrongly formatted cmder link

File: Documentation/Contribution/development_environment.md [hunk 2]
```diff
@@ -2,45 +2,44 @@
...
 
-## Docker
+Choose the editor or IDE you prefer.
 
-Everything is being developed with Docker in mind, and in fact you can get something that resembles the
-production environment pretty easily as things are packaged up and ready to be used.
+### Editor: Visual Studio Code
 
-Download the Community Edition from [here](https://store.docker.com/search?type=edition&offering=community).
+Visual Studio code is a free, open source, cross platform editor:'
```
Comment (anderaus) [inline Documentation/Contribution/development_environment.md]: Extra tick (') at end

File: Documentation/Contribution/getting_started.md [hunk 3]
```diff
@@ -1,14 +1,36 @@
...
+In order for you to get started easily, we have created an overview of the tools needed and how you can get the application up and running. 
 
-_Eager to get started right away? Have a look at the [included example](../../Source/Example/readme.md)._
+If you haven't already familiarized yourself with the [Contributor Guide](./contributing.md), please do so before proceeding. 
+
+_Eager to get started right away? Have a look at the [example application](../../Source/Example/readme.md). to see how each project is structured._
+
+## Setting up your development environment
+
+Make sure you have the required [development tools](./development_tools.md) installed.
```
Comment (anderaus) [inline Documentation/Contribution/getting_started.md]: 404 link

File: Documentation/Contribution/getting_started.md [hunk 4]
```diff
@@ -18,6 +40,3 @@ _Eager to get started right away? Have a look at the [included example](../../So
 
 See [Overview](../Continuous Integration/overview.md) of Continuous Integration for more information.
```
Comment (anderaus) [inline Documentation/Contribution/getting_started.md]: I don't see why, but this isn't output as a correct link for some reason

**Comment by anderaus** [comment]
Oops, just realized I've used the "add single comment" feature instead of the proper github PR review functionality. Sorry about that! Will do it the proper way next time.

**Comment by karolikl** [comment]
No problem, thanks for the thorough review! The last comment about the invalid URL is because on the space, adding %20 does the trick.

**Pull request merged**